backend/tests/test_calculator.py

```diff
@@ -1,4 +1,4 @@
-import server.calculator as calculator
+import backend.server.calculator as calculator
```

Comment:
I guess this should work without backend?

Reply:
Done, this indeed works